### DISCUSSION TITLE
Reorder feature for user name input cell.

### DISCUSSION THREAD

**Discussion by cookie777** [DaiSugi01/Queens-game#38] [Q&A]
As for user name input scene,  I want to hear your opinions.
Is it better to have order feature ? (change the order of cell by drag and drop)

![image](https://user-images.githubusercontent.com/41662956/116324886-ce5ac700-a775-11eb-9b8d-bcd58023f793.png)

**Comment by DaiSugi01** [comment]
I don't think the ordering function is necessary.

**Reply by cookie777** [reply]
Got it. Thanks for the reply!
